packages/amplify-graphql-auth-transformer/src/__tests__/resolvers/subscriptions.test.ts

```diff
@@ -105,33 +104,15 @@ describe('subscriptions', () => {
       });
 
       it('renders for iam auth with no admin roles', () => {
-        const adminRolesDisabledExpr1 = generateAuthExpressionForSubscriptions(
-          configFromPartial({
```

Comment:
These three parameterized ways of representing admin roles (which we were verifying had the same effect here) are now collapsed into one, so the comparison between them is simplified.